src/main/java/com/jalasoft/webservice/controller/ImageController.java

```diff
@@ -70,6 +74,7 @@ public Response Convert (@RequestParam("file") MultipartFile file,
                 imageCriteria.setDestinationPath(utils.getPublic());
                 imageCriteria.setExtension(ext);
             } else {
+                logger.error("The cheksum send is not match - Method: " + new Object() {}.getClass().getEnclosingMethod().getName());
```

Comment:
review this line do not overload the 100 character

Reply:
Done